src/Serialization/Upgrades/1.6.0-1.jl

```diff
@@ -0,0 +1,16 @@
+push!(upgrade_scripts_set, UpgradeScript(
+  v"1.6.0-1",
+  function upgrade_1_6_0_1(s::UpgradeState, dict::Dict)
+    # recurse upgrade on containers
+    upgrade_containers(upgrade_1_6_0_1, s, dict)
+
+    # Upgrades
+    if dict[:_type] isa Dict && dict[:_type][:name] == "RationalFunctionField"
```

Comment:
```suggestion
    if dict[:_type] isa Dict && haskey(dict[:_type], :name) && dict[:_type][:name] == "RationalFunctionField"
```
Otherwise the second part of the condition could fail